metriq_gym/dispatch_clops.py

```diff
@@ -18,8 +18,7 @@
 def main():
     args = parse_arguments()
 
-    if args.token:
-        QiskitRuntimeService.save_account(channel="ibm_quantum", token=args.token, set_as_default=True, overwrite=True)
+    QiskitRuntimeService.save_account(channel="ibm_quantum", token=os.environ.get("IBM_QISKIT_API_KEY"), set_as_default=True, overwrite=True)
```

Comment:
Do we want to read it from the .env file directly rather than from the os? Otherwise we'll need to include instructions for folks to remember to source the .env file when there are changes.

Probably the expected UX would be to import the file using python and then pull out the variable?

Reply:
Yes. The `os.environ.get` will obtain things from either the OS or the `.env` file. However, I noticed that in order for that to be the case, we'll need to invoke `load_dotenv()` so that this is put into memory. Updated this in da467fa